static/js/subs.js

```diff
@@ -225,14 +225,11 @@ exports.update_settings_for_subscribed = function (sub) {
 
         button.toggleClass("checked");
         settings_button.text(i18n.t("Unsubscribe"));
-
-        stream_edit.add_me_to_member_list(sub);
     } else {
         exports.add_sub_to_table(sub);
     }
 
-    var active_stream = exports.active_stream();
-    if (active_stream !== undefined && active_stream.id === sub.stream_id) {
+    if (sub.render_subscribers && stream_edit.is_sub_settings_active(sub)) {
```

Comment:
Subscriptions are already render here, so we don't need call for `add_me_to_member_list`. And as this functions was not used in any other codebase, I removed it.